crates/bevy_light/src/directional_light.rs

```diff
@@ -248,6 +248,9 @@ pub fn update_directional_light_frusta(
 /// By default, the atmosphere is rendered with [`SunDisk::EARTH`], which approximates the
 /// apparent size and brightness of the Sun as seen from Earth. You can also disable the sun
 /// disk entirely with [`SunDisk::OFF`].
+///
+/// In order to cause the sun to "glow" and light up the surrounding sky, enable bloom
+/// in your post-processing pipeline by adding a `Bloom` component to your camera.
```

Comment:
I can't make this a doc-link because there's no dependency between bevy_light and bevy_post_process.

Reply:
what we sometimes do is link the website docs instead in this case

Reply:
Yeah, which I don't hate, but IMO this is easy to search for end users.